test/interpreter_functional/test_suites/run_pipeline/esaggs.ts

```diff
@@ -33,54 +33,67 @@ export default function ({
   getService,
   updateBaselines,
 }: FtrProviderContext & { updateBaselines: boolean }) {
+  const supertest = getService('supertest');
   let expectExpression: ExpectExpression;
+
+  const expectClientToMatchServer = async (title: string, expression: string) => {
```

Comment:
I added a function to compare results from client & server. Longer term we might consider updating `helpers.ts` to have `expectExpressionServer` or similar, which would make it even easier to run expressions on the server.